runtime/tests/run_functional_tests.sh

```diff
@@ -46,9 +46,9 @@ echo done.
 echo Running tests...
 if [ -z "$1" ]
   then
-    ../node_modules/protractor/bin/protractor tests/protractor.config.js 2>&1
+    ../node_modules/protractor/bin/protractor tests/protractor.config.js --params.target.url=$1 2>&1
```

Comment:
One more ref to localhost at line 42:
https://github.com/ldimaggi/fabric8-planner/blob/91432f80be713c9259b6b41f3d731b107d921a07/runtime/tests/run_functional_tests.sh#L42